js/index.js

```diff
@@ -1,7 +1,96 @@
+
+Praesent consequat volutpat suscipit. Mauris ut tellus suscipit, facilisis odio sed, porta urna. Duis ac pharetra ligula. In non varius nibh. Curabitur pretium ex quis neque vulputate, vitae consequat enim tincidunt. Nam in nunc in dui tincidunt rhoncus id sit amet orci. Sed efficitur magna sollicitudin, tempor nunc id, volutpat lacus. In ullamcorper egestas ipsum porta feugiat. Suspendisse egestas nec ante non varius. Nam varius lectus sit amet consectetur sodales.
+
+Vivamus quis est massa. Vestibulum lobortis, libero nec fermentum ultrices, sem metus vulputate dolor, eget pharetra elit sapien quis velit. Nullam porta metus at iaculis hendrerit. Aliquam laoreet eros non dui venenatis dignissim. Maecenas quis metus quis arcu lobortis tempor. Sed lobortis nisi quis ligula dictum, nec consequat nisl dictum. Nullam suscipit porttitor nibh, eget varius lacus. Integer velit sapien, finibus in lorem quis, pellentesque venenatis quam. Quisque purus nunc, tincidunt et orci non, faucibus ullamcorper libero. Morbi euismod ac mauris vel ullamcorper. Aenean quis facilisis velit, vel laoreet diam. Nunc tincidunt est id nibh blandit, eget egestas mi pellentesque. Morbi accumsan et sem sed tempus. Nullam suscipit massa ac pharetra bibendum.`;
+
+loremIpsumSplit = loremIpsum.split(" ");
+console.log(`Amount of words in text: ${loremIpsumSplit.length}`);
+
+etFullCount = 0;
+etCount = 0;
+
+for (let i = 0; i < loremIpsumSplit.length; i++) {
+  if (loremIpsumSplit[i] === "et") {
+    etFullCount++;
+  }
+}
+
+for (let i = 0; i < loremIpsumSplit.length; i++) {
+  if (loremIpsumSplit[i].indexOf("et") > 0) {
+    etCount++;
+  }
+}
+
+console.log(`Full word \'et\' present in text: ${etFullCount}`);
+console.log(`Total amount of pieces of the word \'et\' present in the text: ${etCount}`);
+
+// -----------------------------------------------
+// Bonus 2
+
+phraseToCheck = "A man, a plan, a canal, Panama";
+firstPhrasePartClean = "";
+reversePhrasePart = "";
+
+//firstPhrasePart = phraseToCheck.replace(/,/g,'').toLowerCase().replace(/ /g,'')
+//console.log(firstPhrasePart)
+
+firstPhrasePart = phraseToCheck.toLowerCase().split(" ");
+
+for (let i = 0; i < firstPhrasePart.length; i++) {
+  firstPhrasePart[i].indexOf(",") > 0
+    ? (firstPhrasePartClean += firstPhrasePart[i].substring(0, firstPhrasePart[i].length - 1))
+    : (firstPhrasePartClean += firstPhrasePart[i]);
+}
 
+for (let i = firstPhrasePartClean.length - 1; i >= 0; i--) {
+  reversePhrasePart += firstPhrasePartClean[i];
+}
 
-// Iteration 3: Loops
\ No newline at end of file
+if (firstPhrasePartClean === reversePhrasePart) {
+  console.log("It's a palindrome!");
+} else {
+  console.log("Too bad.. no palindrome...");
+}
```

Comment:
Well done! Really great that you made even the bonus 👏🏻
Now, You just learned about functions! It would be amazing if you could practice this knowledge in this exercise, you would see how many codes you will not repeat.
Go #DRY